src/status_im/subs.cljs

```diff
@@ -1631,3 +1631,10 @@
  :<- [:search/filter]
  (fn [[chats search-filter]]
    (apply-filter search-filter chats extract-chat-attributes)))
+
+;; INTRO WIZARD
+(re-frame/reg-sub
```

Comment:
top level subs should be put at the top of the file

Reply:
Done